mirdata/datasets/fma_keys.py

```diff
@@ -0,0 +1,280 @@
+"""
+FMA Keys Dataset Loader
+
+.. admonition:: Dataset Info
+    :class: dropdown
```

Comment:
alright, now the docstrings looks quite good. However, you need to update some files in the documentation source code to make this visible to users.

- [ ] `docs/source/mirdata.rst`: here, you need to add your dataset as it is done for the rest!
- [ ] `docs/source/table.rst`: here, you need to add your dataset as it is done for the rest as well. Add the number of tracks, available annotations, data availability, licensing, etc.
- [ ] `docs/source/quick_reference.rst`: if your dataset has some annotation type of information that is not available in quick reference to link, please add it there, and therefore, users will be able to click the annotations in the table and read more info!